editor/editor_node.h

```diff
@@ -164,6 +164,7 @@ class EditorNode : public Node {
 private:
 	friend class EditorSceneTabs;
 	friend class SurfaceUpgradeTool;
+	friend class EditorRunBar;
```

Comment:
Seems unnecessary.

Reply:
This is needed on EditorRunBar to call `trigger_option`, so I can make it public instead then

Reply:
But it's already public.

Reply:
Huuum, then there might have been some extra stuff I needed from it before but after code cleanup it isn't needed anymore, I'll recheck it.

Reply:
Fixed. It was the `MenuOption` enum that was private. So, since `trigger_option` is public, I exposed the enum as well.